### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by alebianchin**
Added prerequisites and short tutorials on how to install them on both Linux and Mac Os. 

**Any side note on the changes made:** Also updated some outdated info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ For more information, see the [Eclipse project proposal](http://www.eclipse.org/
...
 -------------------
 
-Eclipse Kura requires at least Java 8 and OSGi R6. All bundles can make use of a Java 8 as execution environment. All bundles must be limited to the "compact2" profile of Java.
+Eclipse Kura is compatible with Java 8 (`Bundle-RequiredExecutionEnvironment: JavaSE-1.8`) and OSGi R6.
 
 
 Development Model
 -----------------
-Development on Kura follows the [gitflow model](http://nvie.com/posts/a-successful-git-branching-model/).  Thus, the working copy is in the develop branch, and the master branch is used for releases.
+Development on Kura follows a [variant of the gitflow model](https://github.com/eclipse/kura/wiki/New-Kura-Git-Workflow).  Development is made on the [develop branch](/eclipse/kura/tree/develop). The master branch is not used anymore.
```
Comment (pierantoniomerlino) [inline README.md:34]: Not sure about the link to the develop branch, but let's check after the merge...

File: README.md [hunk 2]
```diff
@@ -39,6 +39,44 @@ Getting Started
 Development for Kura can be done in Eclipse IDE using the Kura Development Environment, in a gateway or in a Docker container.
 
 ## Development Environment
+
+### Supported Development Platforms
+The Eclipse Installer based setup works for the main used platforms like Linux, Mac Os and Windows.
+
```
Comment (pierantoniomerlino) [inline README.md:45]: Mac OS

File: README.md [hunk 3]
```diff
@@ -39,6 +39,44 @@ Getting Started
...
+### Prerequisites
+Before installing Kura, you need to have the following programs installed in your OS
```
Comment (pierantoniomerlino) [inline README.md]: Before installing Kura, you need to have the following programs installed in your **system**

**Review by pierantoniomerlino — APPROVED** [review]

**Comment by pierantoniomerlino** [comment]
Since you are updating the README file, you'd also modify the Eclipse IDE section in this way:

- remove the "Switch to advanced mode" point
- append to the point "Update Eclipse Kura Git repository username and customize further settings if you like (e.g. Root install folder, Installation folder name)" the sentence "To show these options, make sure that the "Show all variables" checkbox is enabled".

**Comment by MMaiero** [comment]
@alebianchin Can you make the changes suggested by @pierantoniomerlino?
Thanks

**Comment by alebianchin** [comment]
Should be all fixed now!

**Pull request merged**